python/private/python_bootstrap_template.txt

```diff
@@ -461,6 +461,9 @@ relative_files = True
     kparams['stdout'] = sys.stderr
     kparams['stderr'] = sys.stderr
 
+  # Prevent coveragepy report issues when packages execute files in tmp dir.
+  params.append('--omit="$TMPDIR/*"')
```

Comment:
subprocess.call won't exapnd environment variables. So something like this:

You want `tempfile.gettempdir()`
```suggestion
  import tempfile
  omit_pattern = os.path.join(tempfile.gettempdir(), "*")
  params.append(f'--omit={omit_pattern}')
```

Reply:
Ahhh brilliant! Cheers @rickeylev !